mkdocs.yml

```diff
@@ -84,7 +82,7 @@ theme:
   custom_dir: forge_theme
 
 extra:
-  current_version: 1.14.x
+  current_version: 1.16.x
```

Comment:
Not sure what this defines but I asked tterrag earlier about the version that shows up when you open the docs and he said it should be the LTS version. So if this is that, it should be 1.15.x

Reply:
If that's what it is, I'll update date it to LTS instead of current.